doc/rofilib/configuration.rst

```diff
@@ -1,11 +1,54 @@
 Configuration
 =============
 
-Here you are supposed to say the general idea behind the configuration
-representation and also list its capabilities. (e.g., you create and instance,
-put modules, then you have to compute the matrix, etc.)
+Configuration is a way of representing any bot in an universal manner. It
+contains all building-blocks of the RoFI platform such as various modules
+and joints. Using these blocks, you can specify every rofibot and work
+with it. Configuration computes positions of every component within the bot
+and provides useful functions for manipulation with such rofibot, including
+some validity checks (e.g., collision checks).
 
-Feel free to add code examples.
+Using the configuration is pretty straightforward. The main class is
+:cpp:class:`Rofibot <rofi::configuration::Rofibot>` for which you create an
```

Comment:
`:cpp:` is default for our documentation, so you don't need to specify it (but it's fine to leave it)